app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= stylesheet_link_tag 'style.css' %>
+
+<% if user_signed_in? %>
+  <nav class="navbar navbar-light bg-light">
+    <div class="d-inline">
+      <a class="navbar-brand"><%=current_user.organisation.name + " | " + current_user.first_name + " " +current_user.last_name %></a>
+    </div>
+    <div class="form-inline">
+      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+      <div class="dropdown">
+        <button style="margin-left: 0.2cm;" class="btn btn-primary btn-circle btn-md" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><%=current_user.first_name.slice(0,1)+current_user.last_name.slice(0,1) %></button>
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" ><i class="fas fa-user-circle"></i><%=" " + current_user.first_name + " " +current_user.last_name %></a>
```

Comment:
@apeksha2697  please use string interpolation here, like this
"#{current_user.first_name}  #{current_user.last_name}"
or we can write a method in home helper, named full_name and inside that write this piece of code and can call that full_name method here